raw_templates.go

```diff
@@ -1,3 +1,3 @@
\\033[96mWarning:\\033[0m Please install Docker: https://docs.docker.com/docker-for-mac/install/\"\n  exit 1\nfi\n\nif ! type \"docker-compose\" > /dev/null 2>&1; then\n  echo \"\\033[96mWarning:\\033[0m Please install Docker Compose: https://docs.docker.com/docker-for-mac/install/\"\n  exit 1\nfi\n\nif ! type \"php\" > /dev/null 2>&1; then\n  echo \"\\033[96mWarning:\\033[0m Please install PHP, one method is 'brew install php'\"\n  exit 1\nfi\n\nif ! type \"composer\" > /dev/null 2>&1; then\n  echo \"\\033[96mWarning:\\033[0m Please install Composer, https://getcomposer.org/download/\"\n  exit 1\nfi\n\ncomposer install\n\nif test -f whippet.json 2>&1; then\n  if test -f whippet.lock 2>&1; then\n    vendor/bin/whippet deps install\n  else\n    vendor/bin/whippet deps update\n  fi\nfi\n", "templates/script/console.tmpl": "#!/bin/sh\nset -e\n\nexec docker-compose exec wordpress bash\n", "templates/script/server.tmpl": "#!/bin/sh\nset -e\n\ndocker-compose down --remove-orphans\ndocker-compose up\n", "templates/script/setup.tmpl": "#!/bin/sh\nset -e\n\nscript/bootstrap\ndocker-compose exec wordpress /usr/src/app/setup/internal.sh\n", "templates/script/update.tmpl": "#!/bin/sh\nset -e\n\nscript/bootstrap\n", "templates/setup/external.sh.tmpl": "#!/bin/sh\nset -e\n\n##\n## This code will be run during setup, OUTSIDE the container.\n##\n## Because `whippet` running inside the container wouldn't be able to connect\n## to private repositories.\n##\n\nif test -f whippet.json; then\n  whippet deps install\nfi\n", "templates/setup/internal.sh.tmpl": "#!/bin/sh\nset -e\n\n##\n## This code will be run during setup, INSIDE the container.\n##\n\n##############\n#\u00a0Config\n##############\ntitle=\"Your site title here\"\ntheme=your-theme-slug\nplugins=\"a-space-separated list-of plugins-to-activate\"\ncontent=/usr/src/app/setup/content\n\nwp core {{.InstallType}} --skip-email --admin_user=admin --admin_password=admin --admin_email=admin@localhost.invalid --url=http://localhost --title=\"$title\"\n\nfor plugin in $plugins\ndo\n  if wp plugin is-installed $plugin\n  then\n    wp plugin activate $plugin {{.ActivationType}}\n  else\n      echo \"\\033[96mWarning:\\033[0m Plugin '\"$plugin\"' could not be found. Have you installed it?\"\n  fi\ndone\n\nif wp theme is-installed $theme\nthen\n  {{.ThemeEnable}}\n  wp theme activate $theme\nelse\n  echo \"\\033[96mWarning:\\033[0m Theme '\"$theme\"' could not be found. Have you installed it?\"\nfi\n\nimport() {\n  for file in $content/*.xml\n  do\n    echo \"Importing $file...\"\n    wp import $file --authors=skip\n  done\n}\n\nif [ \"$(ls -A $content)\" ]\nthen\n  if wp plugin is-installed wordpress-importer\n  then\n    wp plugin activate wordpress-importer\n    import\n  else\n    echo \"WordPress Importer not installed... installing now\"\n    wp plugin install wordpress-importer --activate\n    import\n    wp plugin uninstall wordpress-importer --deactivate\n  fi\nelse\n  echo \"No content to be imported\"\nfi\n"}
```

Comment:
Do we need to be committing this at all? Can we generate this file at runtime?

Reply:
I assume you mean at install time?

If we did that it wouldn't be possible to install with `go get github.com/dxw/wpc`.

Generated files tend to be part of the repo anyway. For example `.c` (C) files that have been generated from `.yy` (yacc) files.

Reply:
Ah. Ok. Let's label it as generated using a comment, then?

Reply:
@erbridge Pushed another commit. Ready for another look.